CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.5.0] - 2018-10-25
```

Comment:
This was already released?

Reply:
No sorry, I had not paid attention to the base format of the changelog.